flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/LazyFromSourcesSchedulingStrategyTest.java

```diff
@@ -351,9 +354,13 @@ private LazyFromSourcesSchedulingStrategy startScheduling(TestingSchedulingTopol
 	}
 
 	private void assertLatestScheduledVerticesAreEqualTo(final List<TestingSchedulingExecutionVertex> expected) {
-		assertEquals(
-			idsFromVertices(expected),
-			idsFromDeploymentOptions(testingSchedulerOperation.getLatestScheduledVertices()));
+		final List<List<ExecutionVertexDeploymentOption>> deploymentOptions = testingSchedulerOperation.getScheduledVertices();
+		assertThat(expected.size(), Matchers.lessThanOrEqualTo(deploymentOptions.size()));
```

Comment:
We normally add the static import when using matchers.

Reply:
OK. I will do it along with rebasing this change.